docs/sphinx/users/comlinetools/formatlist.txt

```diff
@@ -2,34 +2,35 @@ List supported file formats
 ===========================
 
 A detailed list of supported formats can be displayed using the
-:command:`formatlist` command.
+:program:`formatlist` command.
 
-Current usage information can be shown by running:
+.. program:: formatlist
 
-::
+The default behavior is to print a plain-text list of formats::
 
-    formatlist -help
+    formatlist
 
-The default behavior is to print a plain-text list of formats:
+.. option:: -txt
 
-::
+    This can also be accomplished using by specifying the :option:`-txt`
+    argument::
 
-    formatlist
+        formatlist -txt
 
-This can also be accomplished using by specifying the ``-txt`` argument:
+.. option:: -html
 
-::
+    This option allows to print the list of formats as HTML::
 
-    formatlist -txt
+        formatlist -html
 
-The same information also be formatted as HTML:
+.. option:: -xml
 
-::
+    This option allows to print the list of formats as XML::
 
-    formatlist -html
+        formatlist -xml
 
-or XML:
+.. option:: -help
 
-::
+    Current usage information can be shown by running::
```

Comment:
"Current" seems a bit odd, unless there's a risk of my thinking that it will show me usage information from the past or the future, but I don't strongly object if you like it.